compose-builder/.gitignore

```diff
@@ -2,3 +2,4 @@
 docker-compose.yml
 # staging directory is generated by the shell script gen_secure_compose_ext.sh
 gen_ext_scty/
+res/
```

Comment:
FYI, added for development ability to volume mount in res files without them getting committed